src/diffusers/models/unets/unet_3d_blocks.py

```diff
@@ -1493,7 +1492,7 @@ def __init__(
                 TransformerTemporalModel(
                     num_attention_heads=temporal_num_attention_heads,
                     in_channels=out_channels,
-                    norm_num_groups=temporal_norm_num_groups,
+                    norm_num_groups=resnet_groups,
```

Comment:
This is because in DownBlockMotion, we make use of `resnet_groups` too and `temporal_norm_num_groups` does not seem to be present there. I think this is correct but not too sure.

Reply:
Let's not touch the default values here please especially when we're not too sure. Will defer to @DN6 to verify.

Reply:
Yeah this change should be safe. I think it was a mistake here passing in `temporal_norm_num_groups`

Reply:
awesome! What are your thoughts on the first failing test after these changes? If I make any changes there, multiple other tests are most likely going to break. We are trying to load hf-internal-testing/tiny-stable-diffusion-pipe weights into a different model definition due to the changes in unet...

Reply:
That's okay. We can make a change to that model repo